docs/osquery/osquery.asciidoc

```diff
@@ -82,23 +82,26 @@ You can run packs as live queries or schedule packs to run for one or more agent
 
 . Click the **Packs** tab.
 . Click **Add pack** to create a new pack, or click the name of an existing pack, then **Edit** to add queries to an existing pack.
+. Provide a name for the pack. The short description is optional. 
+. Schedule the pack to be deployed on specified agent policies (*Policy*) or on all agent policies (*Global*).
++
+TIP: Pack deployment details are stored within the <<osquery-custom-config,Osquery configuration>>. The `shard` field value shows the percentage of agents within the policy to which the pack has been deployed.  
++
+If you choose the *Policy* option, you must also configure one or both of the following fields: 
++
+NOTE: When defining pack deployment details, you cannot configure the same policy multiple times. In other words, after specifying a policy, you can either choose to deploy the pack to all of the policy's agents or only a subset. You cannot choose both.
```

Comment:
This note interrupts the flow from the lead-in above and the options below -- is there some other place it could go?

Reply:
Added to the previous line. Lemme know if it's still not flowing and I'll reiterate

Reply:
@nastasha-solomon I think the note is still in the same place? From the latest diff it looks like it's still between the lead-in "configure these fields" and the two bulleted options.